### DISCUSSION TITLE
New line in a record cell [Jun 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2372] [Discord Threads]
> 🙋‍♂️ **johndoe**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### I assume you intended to ask a way to insert new line in a cell.   
####    
#### Multi line text is allowed only when you select data-type as `Long Text`. Use `Shift + Enter` with in a cell of type Long Text to add a new line.  
![image0](https://cdn.discordapp.com/attachments/983230830388322314/983244144761770023/Screenshot_2022-06-06_at_11.10.02_AM.png)  
#  
> 🙋‍♂️ **johndoe**  
#### <@857998597474484295> Yes!  thank you 👍  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983230830388322314)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
